spk/ntopng/Makefile

```diff
@@ -0,0 +1,46 @@
+SPK_NAME = ntopng
+SPK_VERS = 3.8.1
+SPK_REV = 1
+SPK_ICON = src/$(SPK_NAME).png
+
+DEPENDS = cross/$(SPK_NAME)
+
+SPK_DEPENDS = "redis"
+
+# toolchains of DSM <=5.2 have outdated compiler
+REQUIRED_DSM = 6.1
+
+MAINTAINER = hgy59
+DESCRIPTION = ntopng is the next generation version of the original ntop, a network traffic probe that monitors network usage. ntopng is based on libpcap and it has been written in a portable way in order to virtually run on every Unix platform, MacOSX and on Windows as well.
```

Comment:
Is it worth to mention it is "Community Edition", like in your PR description?

Reply:
The description on ntop.org says:

> **Available Versions**
> ntopng comes in three versions, Community, Professional (Small Business Edition) and Enterprise. The Community version is free to use and opensource (code can be found on Github). The Professional and Enterprise offer some extra features that are particularly useful for SMEs or larger organizations. Features are highlighted in the following table.

So IMO it is not needed to explicitly declare this package as "Community Edition", since only this is opensource.

_When I was developping the package, the german magazin `c't` had a series about ntopng, and for raspberry pi there are Pro-Versions for Embedded ARM available (payable)._